docs/manual/cluster-admin/troubleshooting.md

```diff
@@ -67,6 +67,15 @@ If you cannot use GPU in your job, please check the following items on the corre
 
 If the GPU number shown in webportal is wrong, check the [hivedscheduler and VC configuration](./how-to-set-up-virtual-clusters.md).
 
+### NvidiaSmiEccError
+This is a kind of alert from alert manager.
+It means that nvidia cards from the related nodes have double ecc error.
```

Comment:
NvidiaSmiEccError will be fired when there is double/single ECC error, we only crodon the node when there is `double` ECC error